packages/devtools_app/lib/src/screen.dart

```diff
@@ -164,5 +171,12 @@ bool shouldShowScreen(Screen screen) {
       return false;
     }
   }
+  if (screen.requiresVmDeveloperMode) {
+    // TODO(bkonyi): this might be a bit brittle. Should have a global getter
+    // somewhere for 'vmDeveloperModeEnabled'.
+    if (!VmServicePrivate.enablePrivateRpcs) {
+      return false;
+    }
```

Comment:
we could potentially make PreferencesController a global (see globals.dart and `setGlobal`), calling `setGlobal` at the end of `PreferencesController.init()`

Reply:
Good call, didn't realize that was a thing. Done.